api/v1beta1/access_control_validate.go

```diff
@@ -51,6 +52,12 @@ var PredefinedRoles = map[string]struct{}{
 	"write":          {},
 }
 
+var Post6PredefinedRoles = map[string]struct{}{
```

Comment:
I think Roles should be all roles including 6.0 roles. Validation should fail if version < 6.0 and post6.0 role is configured. Simpler is code to use Roles map as being the complete set in other code post validation instead of looking at 2 arrays and possibly more in the furure.

Reply:
Fixed